common/network-common/src/test/java/org/apache/spark/network/sasl/SparkSaslSuite.java

```diff
@@ -362,7 +345,7 @@ public void testDelegates() throws Exception {
     }
   }
 
-  private static class SaslTestCtx {
+  private static class SaslTestCtx implements AutoCloseable {
```

Comment:
Let `SaslTestCtx` implement the `AutoCloseable`, then we can use `try-withresources` in test code